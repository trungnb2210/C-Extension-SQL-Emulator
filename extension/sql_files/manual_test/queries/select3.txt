SELECT size, count
FROM things