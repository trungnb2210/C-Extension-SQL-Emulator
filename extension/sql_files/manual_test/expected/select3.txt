QUERY RESULT:
  size   count 
----------------
| 100  | 2     | 
| 120  | 5     | 
| 300  | 1     | 
----------------
